nncf/quantization/algorithms/weight_compression/weight_lowering.py

```diff
@@ -608,9 +612,11 @@ def _calculate_integer_quantized_weight(
     return compressed_weights
 
 
-def _can_run_optimized(input_backend: TensorBackend) -> bool:
+@cache
```

Comment:
Is it really need to use cache for short function?
And may be will be better to use `tensor` as argument?

Reply:
Caching was suggested by @ljaljushkin https://github.com/openvinotoolkit/nncf/pull/3369#discussion_r2081931851

In order to make it work I had to replace tensor argument with size and backend arguments.

@ljaljushkin what do you think?

Reply:
os.enveron is initialized only ones and access to variable is like to access to a dict
What a problem does using cache solve?

Reply:
@AlexanderDokuchaev, @nikita-savelyevv yes, it's useless
https://docs.python.org/3/library/os.html
![image](https://github.com/user-attachments/assets/cd4bd93c-501e-47d2-b844-01fcd1255310)

Reply:
Removed caching